images/linux/scripts/installers/rust.sh

```diff
@@ -14,8 +14,7 @@ export CARGO_HOME=/usr/share/rust/.cargo
 
 curl https://sh.rustup.rs -sSf | sh -s -- -y
 
-# Add Cargo and Rust binaries to the machine path
-echo "PATH=${CARGO_HOME}/bin:$PATH" | tee -a /etc/environment
```

Comment:
I think the verification at line 32 below is going to fail now since the PATH isn't updated, is that true?

Reply:
@alepauly,
This `source $CARGO_HOME/env` command imports PATH variable from the `$CARGO_HOME/env` , file content:
`export PATH="/usr/share/rust/.cargo/bin:$PATH"`